views/partials/header.hbs

```diff
@@ -1,19 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <title>
-    {{#if publication}}
-     Publication: {{publication.title}} - Octopus
-    {{else}}
-      {{#if problem}}
-      Problem: {{problem.title}} - Octopus
-      {{else}}
-       Octopus. Built for Scientists.
-     {{/if}}
-   {{/if}}
+    {{customTitleTag}}
```

Comment:
Can we still have the else condition with a default fallback?

Reply:
@vot the default fallback is assigned server-side in **routes/middleware.js** line 26